src/platform/plugins/shared/esql/public/triggers/esql_controls/control_flyout/shared_form_components.tsx

```diff
@@ -318,6 +339,73 @@ export function ControlWidth({
   );
 }
 
+export function ControlSelectionType({
+  singleSelect,
+  onSelectionTypeChange,
+}: {
+  singleSelect: boolean;
+  onSelectionTypeChange: (isSingleSelect: boolean) => void;
+}) {
+  const theme = useEuiTheme();
+  const {
+    services: { docLinks },
+  } = useKibana<ServiceDeps>();
+  const multiValuesGuideLink = docLinks.links.query.queryESQLMultiValueControls;
+  return (
+    <>
+      <EuiSpacer size="m" />
+      <EuiFormRow
+        label={i18n.translate('esql.flyout.selectionType.label', {
+          defaultMessage: 'Selections',
+        })}
+        fullWidth
+      >
+        <EuiRadioGroup
+          compressed
+          options={selectionTypeOptions}
+          idSelected={singleSelect ? 'single' : 'multi'}
+          onChange={(id) => {
+            const newSingleSelect = id === 'single';
+            onSelectionTypeChange(newSingleSelect);
+          }}
+          name="selectionType"
+          data-test-subj="esqlControlSelectionType"
+        />
+      </EuiFormRow>
+      {!singleSelect ? (
+        <>
+          <EuiSpacer size="m" />
+          <EuiCallOut
+            announceOnMount
+            size="s"
+            color="primary"
+            iconType="info"
+            css={css`
+              .euiText {
+                color: ${theme.euiTheme.colors.textPrimary} !important;
+              }
+            `}
+          >
+            <EuiText size="s">
+              <FormattedMessage
+                id="esql.flyout.selectionType.callout"
+                defaultMessage="You must use {mvContainsLink} in your ES|QL query for multi-select controls to work."
+                values={{
+                  mvContainsLink: (
+                    <EuiLink href={multiValuesGuideLink} target="_blank">
+                      MV_CONTAINS
```

Comment:
@florent-leborgne we want here to navigate the users to https://www.elastic.co/docs/explore-analyze/dashboards/add-controls#add-esql-control and specifically in a new section that will describe how they can create multi values controls with MV_CONTAINS.

It should be

```
| WHERE MV_CONTAINS(?values, field)
```

so we need to explain to them that if they want multiple selection filters they have to use the variable (values in my example) with MV_CONTAINS like the example above.

We are going to release this feature first in serverless and at some point at 9.3. 

Can you help me with:

- adding this new section 
- can this section have the serverless label for now to indicate that this is only released there?

Let me know if you need more details

Reply:
Those aren't different options, it's the sequence of actions :D